google/cloud/aiplatform/models.py

```diff
@@ -1570,6 +1638,7 @@ def batch_predict(
         ] = None,
         labels: Optional[dict] = None,
         credentials: Optional[auth_credentials.Credentials] = None,
+        encryption_spec_key_name: Optional[str] = None,
```

Comment:
It seems like `Model.deploy` also needs support.

Reply:
Good catch, adding it.